crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -130,11 +130,19 @@ def check_all_repos
       end
 
       def provided_and_enabled?(feature, platform = nil, arch = nil)
-        provided_with_enabled?(feature, platform, arch, true)
+        provided_with_enabled(feature, platform, arch, true).first
+      end
+
+      def provided_and_enabled_with_repolist(feature, platform = nil, arch = nil)
```

Comment:
I guess not (by convention), as it doesn't return a bool only. that's why i took out the `?`